homeassistant/components/frontend/storage.py

```diff
@@ -35,14 +27,19 @@ async def async_user_store(
     hass: HomeAssistant, user_id: str
 ) -> tuple[Store, dict[str, Any]]:
     """Access a user store."""
-    _initialize_frontend_storage(hass)
```

Comment:
This was unreachable since its always initialized before the websocket commands are registered